diktat-rules/src/main/kotlin/generated/WarningNames.kt

```diff
@@ -132,5 +132,5 @@ object WarningNames {
 
     const val TOO_MANY_CONSECUTIVE_SPACES: String = "TOO_MANY_CONSECUTIVE_SPACES"
 
-    const val ENUMS_SEPARATED: String = "ENUMS_SEPARATED"
```

Comment:
Merge error? Why is enums rule removed?

Reply:
yes, merge error

Reply:
Well, I believe it's still here, please merge correctly.